examples/guestbook/v1beta3/frontend-service.json

```diff
@@ -0,0 +1,16 @@
+{
+  "kind": "Service",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "frontend",
+    "labels": {"name": "frontend"},
```

Comment:
In all other objects, you put the curly braces and labels on separate lines. Could you please follow that convention here, for consistency?